From d1e6f100b511a1c321f8486b97d75fe4b085d0fb Mon Sep 17 00:00:00 2001
From: Rajaram Gudivada <rajaramg@codeaurora.org>
Date: Thu, 5 Sep 2013 06:43:55 -0400
Subject: [PATCH 01/10] Camera: Disable Camera Launcher when both Front and
 Back sensors are not detected

Instead of checking only back sensor, this change checks both Front and Back sensors
at the time of first time boot up and then if no sensors are present, disable all
camera activities/CameraLauncher.

Change-Id: Ic9ebd3140caa8df079fe01553d62e96207f2e159
---
 src/com/android/camera/DisableCameraReceiver.java | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/com/android/camera/DisableCameraReceiver.java b/src/com/android/camera/DisableCameraReceiver.java
index aea6ec5..5237c1d 100644
--- a/src/com/android/camera/DisableCameraReceiver.java
+++ b/src/com/android/camera/DisableCameraReceiver.java
@@ -30,7 +30,7 @@ import com.android.camera.debug.Log;
 // this receiver will be disabled, so it will not run again.
 public class DisableCameraReceiver extends BroadcastReceiver {
     private static final Log.Tag TAG = new Log.Tag("DisableCamRcver");
-    private static final boolean CHECK_BACK_CAMERA_ONLY = true;
+    private static final boolean CHECK_BACK_CAMERA_ONLY = false;
     private static final String ACTIVITIES[] = {
         "com.android.camera.CameraLauncher",
     };
-- 
2.7.4

